### PR TITLE
Kickstart tests for UTC and LOCAL hwclock.

### PR DESCRIPTION
**Pull request by KosiehBarter**
Signed-off-by: Karel Valek <kvalek@redhat.com>

### CODE REVIEW

**Comment by jkonecny12** [comment]
The same as PR https://github.com/rhinstaller/anaconda/pull/184 please remove the '...and' and similar.

**Comment by KosiehBarter** [comment]
Fixed: Removed "FAILED:"

**Comment by jkonecny12** [comment]
Seems ok to me now.

**Comment by clumens** [comment]
Yeah, me too.

File: tests/kickstart_tests/timezoneLOCAL.ks [hunk 1]
```diff
@@ -0,0 +1,39 @@
+#version=DEVEL
+url --url="http://dl.fedoraproject.org/pub/fedora/linux/development/$releasever/$basearch/os/"
+install
+network --bootproto=dhcp
+
+bootloader --timeout=1
+zerombr
+clearpart
```
Comment (bcl) [inline tests/kickstart_tests/timezoneLOCAL.ks]: We should be consistent about using zerombr/clearpart in our kickstarts. Most of them have zerombr and clearpart --all --initlabel (I think zerombr is redundant these days, but it doesn't hurt)

**Comment by KosiehBarter** [comment]
Edited locally, added arguments do cleanpart.
Pushed.

**Pull request closed without merging**